src/main/java/com/amazon/opendistroforelasticsearch/search/asynchronous/plugin/AsynchronousSearchPlugin.java

```diff
@@ -84,7 +84,8 @@
 public class AsynchronousSearchPlugin extends Plugin implements ActionPlugin, SystemIndexPlugin {
 
     public static final String OPEN_DISTRO_ASYNC_SEARCH_GENERIC_THREAD_POOL_NAME = "opendistro_asynchronous_search_generic";
-    public static final String BASE_URI = "/_opendistro/_asynchronous_search";
+    public static final String LEGACY_BASE_URI = "/_opendistro/_asynchronous_search";
```

Comment:
`LEGACY_OPENDISTRO_BASE_URI`, who knows how may legacies we'll introduce ... that's also what we have in https://github.com/opensearch-project/opensearch-plugins/blob/main/UPGRADING.md#rest-apis-backward-compatibility AFAIK

Reply:
Will rename this.